File: C++/116A-Tram.cpp
/*
    Solution to Codeforces problem 116A
    Copyright (c) GeneralMing. All rights reserved.

    https://github.com/GeneralMing/codeforces
*/

#include<iostream>
#include<string>

using namespace std;

int main()
{
    int n, b, a, curr = 0, cap = 0;
    cin >> n;
    for(int i = 0; i < n ; i++)
    {
        cin >> a >> b;
        curr -= a;
        curr += b;
        if(cap < curr)
            cap = curr;
    }
    
    cout << cap << endl;
    return 0;
}

File: Python/116A-Tram.py
"""
    Solution to Codeforces problem 116A
    Copyright (c) GeneralMing. All rights reserved.

    https://github.com/GeneralMing/codeforces
"""

n = int(input())
cap = 0
curr = 0
for i in range(n):
	a, b = input().split(" ")
	a = int(a)
	b = int(b)
	curr -= a
	curr += b
	if(cap < curr):
		cap = curr
print(cap)